src/biogeophys/LunaMod.F90

```diff
@@ -409,11 +411,15 @@ subroutine Update_Photosynthesis_Capacity(bounds, fn, filterp, &
                           
                          chg = vcmx25_opt-vcmx25_z(p, z)
                          chg_constrn = min(abs(chg),vcmx25_z(p, z)*max_daily_pchg)
+                         vcmx_prevyr(p,z) = vcmx25_z(p,z)
                          vcmx25_z(p, z)  = vcmx25_z(p, z)+sign(1.0_r8,chg)*chg_constrn
-                          
+                         vcmx_prevyr(p,z) = (vcmx_prevyr(p,z)+vcmx25_z(p,z))/2.0_r8
+ 
                          chg = jmx25_opt-jmx25_z(p, z)
                          chg_constrn = min(abs(chg),jmx25_z(p, z)*max_daily_pchg)
+                         jmx_prevyr(p,z) = jmx25_z(p,z)
                          jmx25_z(p, z)  = jmx25_z(p, z)+sign(1.0_r8,chg)*chg_constrn 
+                         jmx_prevyr(p,z) = (jmx_prevyr(p,z)+jmx25_z(p,z))/2.0_r8
```

Comment:
Let's talk about this Wednesday.

Reply:
I agree chatting tomorrow will help. I found this to be the simplest way to implement what I wanted here which was a running average, but I'm sure there are more robust ways. My goals was to keep a running average of growing season Jmax and Vcmax so that spring initialization of predicting Jmax and Vcmas was not biased. Surprisingly, the constant value chosen previously highly influences the seasonal simulation of GPP.